Makefile

```diff
@@ -355,7 +376,7 @@ check: ## Dockerized version of make test-no-verify
 	$(DOCKER_GO) "make test-no-verify"
 
 .PHONY: verify-unchanged
-verify-unchanged: ## Verify there are no un-committed changes
+verify-unchanged: bundle-reset ## Verify there are no un-committed changes
```

Comment:
Why run bundle-reset as part of this job ?

Reply:
It doesn't really needed :(